.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -2,4 +2,7 @@ blank_issues_enabled: false
 contact_links:
   - name: Kadena Discord
     url: https://discord.io/kadena
-    about: Contact us on Discord if you're having trouble with the library. We have a specific channel for Kadena.js. "#kadena-js" can be found in the "Buidler Zone".
+    about:
+      Contact us on Discord if you're having trouble with the library. We have a
+      specific channel for Kadena.js. "#kadena-js" can be found in the "Buidler
```

Comment:
```suggestion
      specific channel for Kadena.js. "#kadena-js" can be found in the "Builder
```

Reply:
I think this is correct/intentional 🤷

Reply:
https://www.coindesk.com/markets/2019/02/02/the-history-of-hodl/ :)